src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -203,6 +203,8 @@
       File="$(ArchiveArtifactsTextFile)"
       Lines="@(ArtifactUrls)"
       Overwrite="true" />
+
+    <MakeDir Directories="$(TarballRootDir)packages/prebuilt" />
```

Comment:
Is this necessary?  We have a prebuilt directory now that's copied over so I removed this explicit MakeDir.

Reply:
I removed the prebuilt directory with this change because now there's nothing in it.